dist/stat_data.py

```diff
@@ -311,6 +311,13 @@ def __init__(self, name, desc, flags=''):
             'no_clear,no_scale'),
     TxnStat('txn_pinned_range',
         'transaction range of IDs currently pinned', 'no_clear,no_scale'),
+    TxnStat('txn_pinned_snapshot_range',
+        'transaction range of IDs currently pinned by named snapshots',
+            'no_clear,no_scale'),
```

Comment:
To avoid confusion, "no_clear,no_scale" on the pinned range *does* look right.  Just not the on the following two.